tools/analyse-storage/src/commands.rs

```diff
@@ -0,0 +1,228 @@
+use clap::Parser;
+use near_store::db::RocksDB;
+use near_store::{col_name, DBCol};
+use nearcore::get_default_home;
+use once_cell::sync::Lazy;
+use rayon::prelude::*;
+use rocksdb::{ColumnFamilyDescriptor, Options, DB};
+use std::collections::HashMap;
+use std::path::PathBuf;
+use std::sync::{Arc, Mutex};
+use std::{panic, println};
+use strum::IntoEnumIterator;
+
+static DEFAULT_HOME: Lazy<PathBuf> = Lazy::new(get_default_home);
+
+#[derive(Parser)]
+pub struct AnalyseStorageCommand {
```

Comment:
Make it more versatile by extending the `DatabaseCommand` into subcommand, and having current command being `Analyse` subcommand